pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -943,12 +953,54 @@ def init_meta(self):
         """
         super().init_meta()
         # Add the name of the dispersing element
-        # dispangle and filter1 are not defined for Shane Kast Blue
 
         # Required
         self.meta['decker'] = dict(ext=0, card='HIERARCH ESO INS OPTI3 NAME')
 
-    def order_platescale(self, binning = None):
+    @property
+    def norders(self):
+        """
+        Number of orders observed for this spectograph.
+        """
+        return 11
+
+    @property
+    def order_spat_pos(self):
+        """
+        Return the expected spatial position of each echelle order.
+
+        from pypeit import edgetrace
+        edges = edgetrace.EdgeTraceSet.from_file('MasterEdges_A_1_DET01.fits.gz')
+
+        nrm_edges = edges.edge_fit[edges.nspec//2,:] / edges.nspat
+        slit_cen = ((nrm_edges + np.roll(nrm_edges,1))/2)[np.arange(nrm_edges.size//2)*2+1]
```

Comment:
Provide some context for the provided code, and put it in a code block.

Reply:
done (I think)